src/index.html

```diff
@@ -7,8 +7,34 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo-link">
+        <img src="images/logo.png" alt="Logo">
+      </a>
+      <nav>
+        <ul>
+          <li><a href="#" class="nav__apple is-active">Apple</a></li>
+          <li><a href="#" class="nav__samsung">Samsung</a></li>
+          <li><a href="#" class="nav__smartphones">Smartphones</a></li>
+          <li>
+            <a href="#"
+               class="nav__laptops__computers"
+               data-qa="hover">
+              Laptops & Computers
+            </a>
```

Comment:
```suggestion
            <a 
              href="#"
              class="nav__laptops__computers"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```
[Code Style Rules](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md):
>Keep your attributes correctly formatted